.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+sudo: required
+python:
+  - "3.6"
+# command to install dependencies
+# install gdal
+before_install:
+  - sudo apt-get update -y
+  - sudo apt-get install -y build-essential
+  - sudo apt-get install -y gcc
+  - sudo apt-get install -y gdal-bin
```

Comment:
Is there a way to log out the version of GDAL that this installs, for informational purposes. I don't feel the need to pin this based on what @Hagellach37 has said, but I do think it'd be helpful to look back through tests to see which version it is